dll/win32/comctl32/lang/comctl_Et.rc

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
It might be interesting to send your translation upstream (aka. to Wine).

Reply:
We haven't been syncing translations with Wine for years, so this is strictly optional.

Reply:
Well, if that's the case, I'll probably look into it later, and resolve other issues.

Reply:
ok let's keep Wine aside.